toast-tk-runtime/src/main/java/io/toast/tk/runtime/TestRunner.java

```diff
@@ -14,11 +15,20 @@
 import io.toast.tk.dao.domain.impl.test.block.line.TestLine;
 import io.toast.tk.runtime.block.FatalExcecutionError;
 import io.toast.tk.runtime.block.IBlockRunner;
+import io.toast.tk.runtime.block.TestBlockRunner;
 
-class TestRunner {
+public class TestRunner {
 
 	private static final Logger LOG = LogManager.getLogger(TestRunner.class);
 	
+	private boolean interupted = false;
+
+	private UUID lastBlockId;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "lastBlockId" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)